rbenv.fish

```diff
@@ -2,8 +2,12 @@
 #   rbenv plugin for oh my fish
 
 function init --on-event init_rbenv
+  if not available rbenv; and set -q RBENV_ROOT; and not contains "$RBENV_ROOT/bin" $PATH
```

Comment:
It feels a little redudant, and it works without, so I'm not adamant about keeping it. It just felt a little redundant to add $RBENV_ROOT to the $PATH when the rbenv command is already available.